build.sbt

```diff
@@ -6,14 +6,14 @@ import sbt.Keys.libraryDependencies
 // common settings (apply to all projects)
 organization in ThisBuild := "com.gu"
 version in ThisBuild := "0.2.0"
-scalaVersion in ThisBuild := "2.12.3"
+scalaVersion in ThisBuild := "2.12.8"
 scalacOptions in ThisBuild ++= Seq("-deprecation", "-feature", "-unchecked", "-target:jvm-1.8", "-Xfatal-warnings")
 
 // resolvers += "guardian-bintray" at "https://dl.bintray.com/guardian/sbt-plugins/"
 resolvers += DefaultMavenRepository
 
 val awsSdkVersion = "1.11.596"
-val playVersion = "2.6.7"
+val playVersion = "2.7.4"
```

Comment:
This is used to pull in Play Json, but looks like this is out of sync with the version we're using for Play itself (see my other comment in the plugin file)